src/test/ui/issues/issue-39848.stderr

```diff
@@ -2,7 +2,7 @@ error: expected `{`, found `foo`
   --> $DIR/issue-39848.rs:8:19
    |
 LL |         if $tgt.has_$field() {}
-   |         --                 - help: try placing this code inside a block: `{ foo(); }`
+   |         --                 - help: try placing this code inside a block: `{ ) }`
```

Comment:
This is less than ideal, but the test is to avoid ICE regression, not good output.